### ISSUE TITLE
[BUG] Sorting by rating fails

### ISSUE THREAD

**Issue by LnLcFlx** [IgnisDa/ryot#835]
When sorting by rating, there occurs an error
>We encountered an error. If you recently upgraded the server, you may have to logout and login again. If the error still persists, please create a new issue on [GitHub](https://github.com/ignisda/ryot/issues).

After logging out and clearing cookies the issue still persists.
Try for example on the [demo instance](https://app.ryot.io/media/list/movie?sortBy=RATING).

**Comment by IgnisDa** [comment]
Sorting by rating will no longer fail, but will show erratic behavior. This is blocked by https://github.com/SeaQL/sea-orm/issues/2227.
